web/src/components/molecules/PluginDetailPage/ModalContent.tsx

```diff
@@ -75,24 +115,19 @@ const ModalContent: React.FC<Props> = ({
       onCancel={handleCancel}
       okText={t("Choose")}
       cancelText={t("Cancel")}
-      okButtonProps={{ disabled: !workspaceId || !projectId }}
-      bodyStyle={{ padding: "20px 32px", maxHeight: "582px", overflow: "scroll" }}
-      onOk={() => onOpenPluginInReearth?.(workspaceId, projectId)}>
+      okButtonProps={{ disabled: !workspaceId || !projectId || !plugin }}
+      style={{ padding: "20px 32px", maxHeight: "582px", overflow: "scroll" }}
```

Comment:
```suggestion
      styles={{ body: { padding: "20px 32px", maxHeight: "582px", overflow: "auto" } }}
```
Please follow the description ` /** @deprecated Please use `styles.body` instead */`
And i think overflow auto might be better

Reply:
done